volatility3/framework/interfaces/layers.py

```diff
@@ -468,6 +471,49 @@ def _encode_data(
 
     # ## Read/Write functions for mapped pages
 
+    def _access_virtmap_cache(self, section: Tuple[int, int]) -> Optional[list]:
+        """Checks and loads the virtmap cache.
+
+        The virtmap cache corresponds to a previous _scan_iterator
+        output, typically loaded from a file.
+        Args:
+            sections: sections (start, size) to retrieve from the cache
+        Returns:
+            A list containing mappings for a specific section of this layer"""
+
+        # Check if layer is fully constructed first
+        if self.context.config.get(
```

Comment:
The layer_identifier should be unique ? Example `layer_identifier`s : 

- `volatility3.framework.layers.intel.WindowsIntel32e.layer_name` -> identifies the kernel layer (`TranslationLayerRequirement` name)
- `volatility3.framework.layers.intel.WindowsIntel32e.layer_name_Process5948` -> identifies the process 5948 

I might have missed something, but it shouldn't be possible to have a duplicate layer string identifier in the layers pool ?

This specific "config"/"cache" is intended to be used for a unique memory capture, as even a dump from the same kernel a few seconds later would have different mappings.

Reply:
Layer name is unique for a run of volatility, but they'll likely all say `primary1` or `memory_layer1` or something.  Across runs they're unlikely to even be different.

The process layers, similarly, won't be different across different images that have processes with the same pid...  I don't think a dump from a few seconds later would have a different cache?  The layer name would likely be the same, and many of the process layer names would too, but also, it could match a wildly different image...